backend/utils.py

```diff
@@ -121,6 +121,11 @@ def login(self, *args):
         method(self, user, *args)
     return login
 
+def follow_health_ministry(user,health_ministry):
+    user.follows.append(health_ministry.key)
```

Comment:
You should use the functions of the model, user.follow(..) and institution.follow(...)